remote/remote_state_s3.go

```diff
@@ -285,7 +290,7 @@ func createS3BucketIfNecessary(s3Client *s3.S3, config *ExtendedRemoteStateConfi
 		}
 
 		if shouldCreateBucket {
-			return CreateS3BucketWithVersioning(s3Client, config, terragruntOptions)
+			return CreateS3BucketWithVersioningSSEcryptionAndAccessLogging(s3Client, config, terragruntOptions)
```

Comment:
Ecryption => Encryption?

```suggestion
			return CreateS3BucketWithVersioningSSEncryptionAndAccessLogging(s3Client, config, terragruntOptions)
```

Reply:
@lorengordon Hey Loren - a very nice catch - thank you! Will fix this today.